specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2023-08-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,687 @@
+        "effect"
+      ],
+      "properties": {
+        "effect": {
+          "description": "The effect for rule. `Deny` rules will overrule the `Allow` rules when there are conflicts.",
+          "enum": [
+            "Allow",
+            "Deny"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Decision",
+            "modelAsString": true
+          }
+        },
+        "permission": {
+          "type": "array",
+          "description": "Array of attribute predicates",
+          "items": {
+            "$ref": "#/definitions/AttributePredicate"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "AttributePredicate": {
+      "type": "object",
+      "description": "Attribute predicate for a policy permission",
+      "properties": {
+        "attributeName": {
+          "type": "string",
+          "description": "AttributeName of a policy permission"
+        },
+        "attributeValueIncludedIn": {
+          "type": "array",
+          "description": "values for a policy permission",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SystemData": {
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "createdBy": {
+          "type": "string",
+          "description": "Created by"
+        },
+        "createdByType": {
+          "type": "string",
+          "description": "Created by type"
+        },
+        "createdAt": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Created at"
+        },
+        "lastModifiedBy": {
+          "type": "string",
+          "description": "last modified by"
+        },
+        "lastModifiedByType": {
+          "type": "string",
+          "description": "last modified by type"
+        },
+        "lastModifiedAt": {
+          "format": "date-time",
+          "type": "string",
+          "description": "last modified at"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "api-version": {
+      "in": "query",
+      "name": "api-version",
+      "description": "The api version to use.",
+      "required": true,
+      "type": "string",
+      "x-ms-client-name": "ApiVersion",
+      "x-ms-parameter-location": "client"
+    },
+    "endpoint": {
+      "in": "path",
+      "name": "endpoint",
+      "description": "The endpoint of your Purview account. Example: https://{accountName}.purview.azure.com.",
+      "required": true,
+      "type": "string",
+      "format": "url",
+      "x-ms-parameter-location": "client",
+      "x-ms-skip-url-encoding": true
+    },
+    "policyName": {
+      "in": "path",
+      "name": "policyName",
+      "description": "policy name",
+      "required": true,
+      "type": "string"
+    },
+    "state": {
+      "in": "path",
+      "name": "state",
+      "description": "policy state",
+      "required": true,
+      "type": "string"
+    },
```

Comment:
We may not need it, depends on https://github.com/Azure/azure-rest-api-specs/pull/25059#discussion_r1298063068

If we really need it, I assume it be extensible enum?

Reply:
updated the above comment with details.